HISTORY.md

```diff
@@ -51,7 +51,7 @@ Changes to JGraphT in each version:
     - Added `DinicMFImpl` which implements Dinic's maximum flow algorithm (contributed by Kirill Vishnyakov)
     - Optimized `PushRelabelMFImpl` implementation which drastically improves max flow computations on certain graphs. (contributed by Alexandru Văleanu) 
     - Added `RandomRegularGraphGenerator` (contributed by Emilio Cruciani)
-    - Reimplemented significantly faster version of Prim's minimum spanning tree algorithm using a FibonacciHeap (suggested by Joris Kinable, contributed by Alexandru Văleanu)
+    - Reimplemented significantly faster version of Prim's minimum spanning trees algorithm using a FibonacciHeap (suggested by Joris Kinable, contributed by Alexandru Văleanu)
```

Comment:
Heh, I'm pretty sure you accidentally did a search/replace for tree->trees but set your editor scope too wide :)